content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -39,31 +39,37 @@ model where `Pods` can be treated much like VMs or physical hosts from the
 perspectives of port allocation, naming, service discovery, load balancing,
 application configuration, and migration.
 
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (which is an optional capability). Even if a host have
+multiple IPs, host-network pods only have one Kubernetes IP address at
+the `Pod` scope, so the "IP-per-pod" model is guaranteed.
+
 Kubernetes imposes the following fundamental requirements on any networking
 implementation (barring any intentional network segmentation policies):
 
-   * pods on a node can communicate with all pods on all nodes without NAT
-   * agents on a node (e.g. system daemons, kubelet) can communicate with all
-     pods on that node
+   * pod-network pods on a node can communicate with all pod-network pods on
+     all nodes without NAT
+   * non-pod processes on a node (e.g. system daemons, kubelet) can communicate
+     with all pods on that node
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux):
+Note: For those platforms that support `Pods` running in the host network (e.g.Linux):
 
-   * pods in the host network of a node can communicate with all pods on all
-     nodes without NAT
+   * host-network pods of a node can connect directly with all pods IPs
+     on all nodes, with source NAT if required
```

Comment:
Optionally, use a callout. I don't think it's important to mention the OS that the node is using.

Here's what I'd write:
```suggestion
{{< note >}}
Pods running with `hostNetwork: true` can send traffic towards any Pod, even to
Pods running on other nodes. The network plugin is allowed to use techniques
such as source NAT to achieve this. Kubernetes does not specify what the source
IP address of that traffic would be; however, it should not be from the IP address
range reserved for Pods.
{{< /note >}}
```

Reply:
suggestion make sense, just to clarify that it can be from the range reserved for pods :), actually it   is what happens, since the host use to adopt the .1 of the pod subnet in the node, that is the closest to the pod addresses, so when you create a connection from a hostnetwork pod, the kernel picks that address

Reply:
also is not only network plugin dependent, there are more factors

Reply:
the idea is this:
```
host-network pods can communicate with all pods in all nodes, however, the source IP used by the host-network pod can be any IP in the host and use source NAT if required.
```

Reply:
I want to be careful not to treat this as an "aside".  It's an additional hard requirement.

Reply:
check this approach, I think that NAT is misleading here, the different IPs in a host is the key, NAT is a detail